src/Stratis.Bitcoin/Utilities/MemorySizeFactor.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Stratis.Bitcoin.Utilities
+{
+    public static class MemorySizeFactor
```

Comment:
Maybe this number should be in the class of the offending object?

Reply:
I don't mind really but its also use in the block store.